app/src/main/java/com/example/encoratask/repository/network/PageInfo.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.encoratask.repository.network
+
+class PageInfo (
```

Comment:
Do you really need this data class to be converted from JSON to Kotlin class? If you're not implementing a pager or using any from this data to your process, why do you map it?